### PR TITLE
Add basic ReadMe.md

### PR DESCRIPTION
**Pull request by DanRStevens**

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# OP2Helper
+
+The OP2Helper library is used to ease development of Outpost 2 missions. It does this by wrapping the Outpost2DLL library, and builds functionality on top of the exported methods. While the Outpost2DLL library is aimed at providing the raw exports from Outpost2.exe (and only the raw exports), the OP2Helper library contains custom code to build functionality around the exported methods.
```
Comment (Brett208) [inline ReadMe.md]: Very minor, I think it should read "and building functionality" for the tense to be consistent.

**Review by Brett208 — APPROVED** [review]
One very minor grammar recommendation. Looks good.

File: ReadMe.md (hunk 1 above)
Comment (DanRStevens) [reply ReadMe.md]: I assume you mean the first instance. I'm having trouble seeing this one.

Comment (Brett208) [reply ReadMe.md]: I think it should read:

It does this by wrapping the Outpost2DLL library, and **building** functionality on top of the exported methods

That would keep the tense the same across the sentence.

I would just merge if you don't agree or don't see it. Not worth spending any more time on.

Comment (DanRStevens) [reply ReadMe.md]: You're probably right. That section does seem like it could be reworded and simplified. And yeah, the tense does seem a bit off.

**Comment by DanRStevens** [comment]
How about this? The grammar tense should be more consistent, and a few fluff words have been cut out.

Could probably cut more if I really thought about it.

**Review by Brett208 — APPROVED** [review]
Looks good. Thanks for writing all these readme files.

**Pull request merged**